docs/tutorials/qaoa.ipynb

```diff
@@ -1,24 +1,16 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.    nx.set_edge_attributes(working_graph, {e: {'weight': np.random.randint(0, 500) / 100} for e in working_graph.edges})
</pre><p>I'd keep a version of the comment what this is doing as in the previous notebook, (i.e. "randomly weighted edge as arandom 2 decimal floating point between 0 and 5")</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/5200/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
done